registration/src/main/proto/lobby/registration/order/order.proto

```diff
@@ -45,12 +45,14 @@ message Order {
     // The status of the order.
     enum Status {
         // The initial status of the order.
-        CREATED = 0;
+        NEW = 0;
+        // The order is in this status when it was already created but the reservation was not confirmed yet.
+        CREATED = 1;
         // The order is in this status when the system has temporarily reserved the seats requested by the registrant.
-        RESERVED = 1;
+        RESERVED = 2;
         // The order is in this status when the registrant has successfully paid for order items.
-        CONFIRMED = 2;
-        // The order is in this status when the system cannot reserve the needed quantity of seats requested by the registrant.
-        REJECTED = 3;
+        CONFIRMED = 3;
+        // The order is in this status when it is placed but already expired.
```

Comment:
Can we have an alias for this item? Or have `EXPIRED` if it's the only case of order rejection?

What's the name of the field in Microsoft's implementation?